test/e2e/framework/pods.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"sync"
+
+	"k8s.io/kubernetes/pkg/api"
+
+	. "github.com/onsi/gomega"
+)
+
+// TODO: Consolidate pod-specific framework functions here.
+
+// Create a new pod according to the framework specifications, and wait for it to start.
+func (f *Framework) CreatePod(pod *api.Pod) {
+	f.mungePodSpec(pod)
```

Comment:
I'm not sure exactly what you mean here - are you suggesting calling CreatePod from CreatePods? I wanted to start all the pods in parallel, which is why I didn't do that (the old code would wait for each pod to be running before starting the next)